packages/server/graphql/mutations/helpers/createGcalEvent.ts

```diff
@@ -47,10 +47,12 @@ const createGcalEvent = async (input: Input) => {
   const calendar = google.calendar({version: 'v3', auth: oauth2Client})
   const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}`)
```

Comment:
+1 Going forward we should think about what to do with the participants. I don't like that right now they're separate from your team members, while on the other hand I like that it's easy to bring others into a parabol meeting this way. Going forward we probably should exchange the invitee input field with the one Igor built to allow a mix of existing and new members. In the meantime we should probably replace the meeting link with an invite link to make it easier for folks to get in.

Reply:
> Going forward we probably should exchange the invitee input field with the one Igor built to allow a mix of existing and new members

Yeah, I agree. I really like the input field Igor & Alicia worked on and we should probably add it to all meeting types and think about how it works alongside the gcal scheduler. 

> In the meantime we should probably replace the meeting link with an invite link to make it easier for folks to get in.

If we include the invite link, it would take the users to the most recent meeting, right? But that might not be the correct meeting, which could be confusing.

We could include both the meeting link and the invite link in the gcal description? 

There could be privacy concerns with including the invite link as you might be able to access someone's gcal calendar, but shouldn't access their Parabol team. For example, there's an Exco team that discusses sensitive info that I don't have access to, but I can view everyone's Parabol gcal calendar, so if we include the invite link in the gcal event, I could join that meeting. 

Still, I think that's probably fine. We could include a checkbox in the gcal modal UI to include the invite link. 

I can merge this PR to make sure it lands in the next release and see what the Squad thinks about including the invite link.